packages/mail/package.json

```diff
@@ -28,7 +28,8 @@
   },
   "dependencies": {
     "@sendgrid/client": "^7.2.2",
-    "@sendgrid/helpers": "^7.2.0"
+    "@sendgrid/helpers": "^7.2.0",
+    "deepmerge": "^4.2.2"
```

Comment:
I don't like the idea of adding another dependency to maintain this change. Can this be done without the additional package?

Reply:
I thought on this. Left on the notes. I see this is also used on helpers, and doesn't make sense to export it from there to use it here. I could have used it even without adding it to the dependencies but I think have it explicit is better.
We can either use merge-data-deep from the helpers or not sure what's your approach to use `deepmerge`
Thoughts?

Reply:
It's not an added dependency since it's already included with helpers. Explicitly adding it to the dependencies here is the best practice. I see no issue.